src/views/CreateNewCardCustomPicker.vue

```diff
@@ -295,6 +298,31 @@ export default {
 		openNewCard() {
 			window.location = generateUrl('/apps/deck') + `#/board/${this.selectedBoard.id}/card/${this.newCard.id}`
 		},
+		preSelectBoard() {
+			const selectedBoardId = Number(localStorage.getItem('selectedBoardId'))
```

Comment:
Please prefix the storage keys with deck. similar how we do it in other places:

```
store/main.js
63:		navShown: localStorage.getItem('deck.navShown') === null || localStorage.getItem('deck.navShown') === 'true',
64:		compactMode: localStorage.getItem('deck.compactMode') === 'true',
223:			localStorage.setItem('deck.navShown', navState)
230:			localStorage.setItem('deck.compactMode', state.compactMode)
```

It would actually be nice to switch to https://github.com/nextcloud-libraries/nextcloud-browser-storage which allows scoping  local storage, but I'd be ok to skip that as we'd need to move all places then.

Reply:
Example usages:
https://github.com/nextcloud/notifications/blob/7c48081354bd179bdc3e51bc3b5c5ce556efc022/src/services/BrowserStorage.js#L21C1-L23C77
https://github.com/nextcloud/notifications/blob/7c48081354bd179bdc3e51bc3b5c5ce556efc022/src/services/notificationsService.js#L49